docs/docs/developers/guides/js_apps/pay_fees.md

```diff
@@ -10,27 +10,28 @@ This guide shows the various ways this can be done. For more of a background see
 
 ## Methods to pay a tx fee
 
-### Natively via an existing balance of fee juice
+1. Natively via an existing balance of fee juice
 
 Either your own account or someone else's address has a balance of fee juice, and the transaction is paid for from this balance.
 
-### Natively via a balance of fee juice to be claimed
+2. Natively via a balance of fee juice bridged from L1 to be claimed on L2
 
 Processing this transaction first claims bridged fee juice then is paid for from the balance, all in the same public/private tx.
 
-### Privately or publicly via a Fee Paying Contract
+3. Privately or publicly via a Fee Paying Contract
 
-A fee paying contract (FPC) is created and nominates a token that it accepts to then pay for txs in fee juice. So a user doesn't need to hold fee juice, they only need the token that the FPC accepts.
+A fee paying contract (FPC) is created and nominates a token that it accepts to then pay for transactions in fee juice. So a user doesn't need to hold fee juice, they only need the token that the FPC accepts.
```

Comment:
```suggestion
A fee paying contract (FPC) is created and nominates a token that it accepts to then pay for transactions in fee juice. So a user doesn't need to hold fee juice, they only need the token that the FPC accepts.

The FPC can also decide to refund users in cases of overpayment. For private FPCs, this incurrs a substantial gate count (proving time) and DA costs so in some cases it might be cheaper to not get the refund. The sample FPCs we show here refund the users but a FPC deployer or user should think carefully if asking for refunds is a smart option

```
```